calico_containers/tests/unit/datastore_test.py

```diff
@@ -398,17 +398,17 @@ def test_ensure_global_config(self):
         """
         self.etcd_client.read.side_effect = EtcdKeyNotFound 
         self.datastore.ensure_global_config()
-        expected_writes = [call(CALICO_V_PATH + "/Ready", "true"),
-                           call(CONFIG_PATH + "InterfacePrefix", "cali")]
-        self.etcd_client.write.assert_has_calls(expected_writes,
-                                                any_order=True)
+        expected_writes = [call(CONFIG_PATH + "InterfacePrefix", "cali"),
+                           call(CALICO_V_PATH + "/Ready", "true")]
+        self.etcd_client.write.assert_has_calls(expected_writes)
```

Comment:
I'm not sure how that would look as a UT.  If CONFIG_PATH exists but InterfacePrefix doesn't then the read side effect would be EtcdKeyNotFound as per this UT.

I can change this test to verify that the read is called with CONFIG_PATH+"InterfacePrefix".

One of the STs (the one where the default is set before creating the nodes) tests this with a real etcd backend.